packages/edit-site/src/components/editor/index.js

```diff
@@ -156,6 +174,32 @@ function Editor() {
 		[ page?.context ]
 	);
 
+	let leftSidebar = null;
```

Comment:
Should we wrap this block into helper function or make it a separate component?

Reply:
Yep! I'd say go for a separate component. And if we are already there, we should extract inserter panel as a separate component as well. What do you think about this structure?

```
packages/edit-site/src/components/left-sidebar
packages/edit-site/src/components/left-sidebar/inserter-panel
packages/edit-site/src/components/left-sidebar/navigation-panel (note: NavigationPanel instead of NavigationSidebar)
```

Reply:
Sounds fine. 👍